lib/_http_client.js

```diff
@@ -324,6 +324,9 @@ function emitAbortNT() {
   this.emit('abort');
 }
 
+function emitCloseNT() {
```

Comment:
minor nit: the "NT" suffix is generally used for functions being passed to `process.nextTick()`, it's probably good enough to just call it `emitClose`.